exammode/static/exammode/css/exam.css

```diff
@@ -0,0 +1,50 @@
+/* Add padding to the whole window */
+.col-sm-12 {
+  padding: 0 5% 0 5%;
+}
+
+/* Remove padding and margin from panels */
+.panel-body {
+  padding: 0;
+  margin: 0;
+}
+
+.panel-heading {
+  cursor: pointer;
+}
+
+.contentDivider {
```

Comment:
Still using camelCase in CSS class names, but CSS classes should be named like `content-divider` instead of camelCase.

Reply:
Naming style changed.